src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -994,6 +995,27 @@ TEST_P(OVClassLoadNetworkTest, LoadNetworkWithBigDeviceIDThrows) {
     }
 }
 
+TEST_P(OVClassLoadNetworkWithAutoBatchingTest, LoadNetworkWithAutoBatchingNoThrows) {
```

Comment:
pls use conventional (common) test, there is nothing specific in the ALLOW_AUTO_BATCHING config key, to justify the creation of the new test. 
If you need a new test case, create a common one e.g. CompileNetworkCorrectProperties